src/main/resource-applier/resource-applier.ts

```diff
@@ -129,9 +129,9 @@ export class ResourceApplier {
 
     const args = [
       subCmd,
-      "--kubeconfig", `"${proxyKubeconfigPath}"`,
+      "--kubeconfig", proxyKubeconfigPath,
       ...parentArgs,
-      "-f", `"${tmpDir}"`,
+      "-f", tmpDir,
```

Comment:
I guess this explains this when I try to uninstall Lens metrics? (Just confirmed it)
![Screen Shot 2022-11-17 at 9 31 36 AM](https://user-images.githubusercontent.com/40840436/202473875-156f4ff4-8334-40a3-b61b-6d5b078faa36.png)